Dashboard/va.gov.artemis.commands/Dsio/Observation/DsioSaveObservationToDifferentNamespaceCommand.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using VA.Gov.Artemis.Vista.Broker;
+using VA.Gov.Artemis.Vista.Utility;
+
+namespace VA.Gov.Artemis.Commands.Dsio.Observation
+{
+    /// <summary>
+    /// A command used to save an observation to VistA using an RPC  
+    /// </summary>
+    public class DsioSaveObservationToDifferentNamespaceCommand : DsioCommand
```

Comment:
DifferentNamespaceCommand?  why not call the exact one (CPRS)?  Why are we sticking with the Dsio prefix?